tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/view_image.py

```diff
@@ -16,7 +16,8 @@ class ViewImage(Base):
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == 'Gallery')
+        from gaiatest.app.Gallery.app import Gallery
+        Wait(self.marionette).until(lambda m: self.apps.displayed_app.manifest_url == Gallery.manifest_url)
```

Comment:
Yeah, I'm not sure about the static method, it might not work. If it doesn't, Martijn's solution is the right one.